examples/expressions/11_multiple_tables.py

```diff
@@ -0,0 +1,218 @@
+"""
+
+.. |choose_from| replace:: :func:`skrub.choose_from`
+.. |choose_int| replace:: :func:`skrub.choose_int`
+.. |choose_float| replace:: :func:`skrub.choose_float`
+.. |MinHashEncoder| replace:: :class:`skrub.MinHashEncoder`
+.. |StringEncoder| replace:: :class:`skrub.StringEncoder`
+.. |TableVectorizer| replace:: :class:`skrub.TableVectorizer`
+.. |var| replace:: :func:`skrub.var`
+.. |TableReport| replace:: :class:`skrub.TableReport`
+.. |HistGradientBoostingClassifier| replace::
+   :class:`sklearn.ensemble.HistGradientBoostingClassifier`
+.. |get_randomized_search| replace:: :meth:`skrub.Expr.get_randomized_search`
+
+.. _example_expressions_intro:
+
+
+Building multi-table machine learning pipelines with DataOps
+==================================
```

Comment:
```suggestion
Multiples tables: building machine learning pipelines with DataOps
==================================================================
```
I wanted to put more emphasis on the multiple tables thing